unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CursorController.cs

```diff
@@ -4,31 +4,25 @@
 
 public class CursorController : MonoBehaviour
 {
+    public static CursorController i { get; private set; }
     public Image cursorImage;
     public Sprite normalCursor;
     public Sprite hoverCursor;
 
-    void OnEnable()
+    void Awake()
```

Comment:
Found that a big reason for our `InteractionHoverCanvas` flaky tests were due to `PointerEventsController` being re-initialized after `OnEnabled` is called, losing the event suscriptions. As this class should be at a higher layer of abstraction than `PointerEventsController`, the dependencies were inverted and the problem fixed.